rele/client.py

```diff
@@ -43,14 +43,12 @@ def __init__(self, gc_project_id, credentials, default_ack_deadline=None):
 
         if USE_EMULATOR:
             self._subscriber_client = pubsub_v1.SubscriberClient()
-            self._publisher_subscriber_client = pubsub_v1.PublisherClient()
+            self._publisher_client = pubsub_v1.PublisherClient()
         else:
             self._subscriber_client = pubsub_v1.SubscriberClient(
                 credentials=credentials
             )
-            self._publisher_subscriber_client = pubsub_v1.PublisherClient(
-                credentials=credentials
-            )
+            self._publisher_client = pubsub_v1.PublisherClient(credentials=credentials)
```

Comment:
Even better: let's not initialize a publisher client in the init that the subscription won't use 99.999% of the time :smile: 

We can have a `_create_topic` that initializes a local client just for that purpose

Reply:
Ok! Done 😄